src/BenchmarkDotNet.Core/Portability/RuntimeInformation.cs

```diff
@@ -141,6 +152,33 @@ internal static Jit GetCurrentJit()
             return HasRyuJit() ? Jit.RyuJit : Jit.LegacyJit;
         }
 
+        internal static string GetJitInfo()
+        {
+            if (IsMono())
+                return ""; // There is no helpful information about JIT on Mono
+#if CORE
+            return "RyuJIT"; // CoreCLR supports only RyuJIT
```

Comment:
"CoreCLR supports only RyuJIT" - currently we allow our users to run only x64 benchmark for Core, so this is true today. 

However if we enable  32bit support for .NET Core 1.1 it won't be true, because right now .NET Core is using Legacy Jit for 32bit. And 32bit .NET Core has support for Windows now only.

.NET Core 1.2 will move from leagacy Jitr for 32bits to RyuJIT which will be used by default. Most probably then also other OSes will get 32bit support.

The goal of my comment is just to share the knowledge so we don't forget it about in the future